deploy/helm-chart/README.md

```diff
@@ -111,8 +129,10 @@ The following tables lists the configurable parameters of the NGINX Kubernetes G
 |`nginxGateway.image.tag` | The tag for the NGINX Kubernetes Gateway image. | edge |
 |`nginxGateway.image.pullPolicy` | The `imagePullPolicy` for the NGINX Kubernetes Gateway image. | Always |
 |`nginxGateway.gatewayClassName` | The name of the GatewayClass for the NGINX Kubernetes Gateway deployment. | nginx |
-|`nginxGateway.gatewayControllerName` | The name of the Gateway controller. The controller name must be of the form: DOMAIN/PATH. The controller's domain is k8s-gateway.nginx.org. | k8s-gateway.nginx.org/nginx-gateway-controller |
+|`nginxGateway.gatewayControllerName` | The name of the Gateway controller. The controller name must be of the form: DOMAIN/PATH. The controller's domain is gateway.nginx.org. | gateway.nginx.org/nginx-gateway-controller |
 |`nginxGateway.kind` | The kind of the NGINX Kubernetes Gateway installation - currently, only Deployment is supported. | deployment |
+|`nginxGateway.crdConfig.name` | The name of the NginxGateway CRD for configuring the control plane. | nginx-gateway-config |
```

Comment:
This is only installed per namespace. Do we intend to have multiple NKGs in the same namespace?

Reply:
I would think it should be possible to have multiple NKG deployments in the same namespace managing resources from different GatewayClasses for example.

Reply:
If we autogenerate, I would want it to be an easy name, since we have to document this and users have to be able to easily edit it...

Reply:
I see we use release name for deployment in helm, I guess that should work.

And in our raw manifest, `nginx-gateway-config` could be the default, since `nginx-gateway` is the default deployment name.